### PR TITLE
Improve get started/installation docs

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR replaces the document at `/docs/get/README.md`, with the goal of improving the installation process by removing potentially confusing references to Netdata Cloud and claiming. It also improves visibility of the kickstart/Docker methods, while also using a `<details>` section to open up a list of all the available methods.

I'm working on new/improved content around the dashboard and chart interactions. Part of that effort will include moving the content in the **Open and explore the dashboard** and **Interact with charts** sections into different documents.

##### Component Name

area/docs

### CODE REVIEW

File: docs/get-started.mdx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+      to="/docs/agent/packaging/installer/methods/cloud-providers"
+      os="Cloud providers (GCP, AWS, Azure)" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/freebsd"
+      os="FreeBSD"
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/pfsense"
+      os="pfSense" 
+      community />
```
Comment (OdysLam) [inline docs/get-started.mdx:105]: In the drop down for installation boxes, we need a line that defines the "community-supported". 

Also in the installation docs we need to link to the original packages and maintainer name so they know where to go in-case it doesn't work. I suggest we remove all these and we tackle them properly in a different repo.

Let's do it right.

File: docs/get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+## Install Netdata on Linux
+
+The **recommended** way to install Netdata on a Linux node (physical, virtual, container, IoT) is our one-line
+[kickstart script](/packaging/installer/methods/kickstart.md). This script automatically installs dependencies and
+builds Netdata from its source code.
+
+Copy the script, paste it into your node's terminal, and hit `Enter` to begin the installation process.
+
+<OneLineInstall />
```
Comment (OdysLam) [inline docs/get-started.mdx:129]: Let's make the "anonymous statistics" a link that links to the page that defines it. ( I think it's security, under a header).

File: docs/get-started.mdx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+![Animated GIF of the standard Netdata dashboard being manipulated and synchronizing
+charts](https://user-images.githubusercontent.com/1153921/80839230-b034a800-8baf-11ea-9cb2-99c1e10f0f85.gif)
+
+Let's cover two of the most important ways to interact with charts: panning through time and zooming.
+
+To pan through time, **click and hold** (or touch and hold) on any chart, then **drag your mouse** (or finger) to the
+left or right. Dragging to the right pans backward through time, while dragging to the left pans forward in time. Think
+of it like pushing the current timeframe off the screen so you can see what came before or after.
+
+To zoom, press and hold `Shift` or `Alt`, then use your mouse's scroll wheel, or a two-finger pinch if you're using a
```
Comment (OdysLam) [inline docs/get-started.mdx:188]: In MacOS it's only Shift. Should we give instructions per OS (Windows, linux, macos) ?

**Review by OdysLam — APPROVED** [review]

File: docs/get-started.mdx (hunk 2 above)
Comment (joelhans) [reply docs/get-started.mdx:129]: Got this going on the preview: https://deploy-preview-534--netdata-docusaurus.netlify.app/docs/get-started/#install-on-linux-with-one-line-installer-recommended

**Review by OdysLam — APPROVED** [review]

**Review by schumacr1 — APPROVED** [review]

**Pull request merged**